ImageFilterEditor/ImageFilterEditor/Filtering/Models/ImageFilterLinearControl.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  FilterControl.swift
+//  ImageFilterEditor
+//
+//  Created by Shawn Gee on 5/5/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
```

Comment:
Nice, I really like this; can allow for a very simple & "Swifty" interface for setting up your filters.

Reply:
This sort of follows an MVVM type of architecture in a way; since this struct won't actually be doing the control of the filter, I wonder if this is the right name for the entity (unless you wanted to make it into a view with SwiftUI)? Maybe something like `ImageFilterLinearControlData`? or `ImageFilterLineraControlModel`? That starts getting pretty darn verbose, though.